core/src/components/toggle/toggle.ionic.scss

```diff
@@ -0,0 +1,138 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./toggle.common";
+@import "./toggle.ionic.vars";
+
+// Ionic Toggle
+// -----------------------------------------
+
+:host {
+  --track-background: #{$toggle-ionic-background-color-off};
+  --track-background-checked: #{ion-color(primary, base)};
+  --border-radius: #{$toggle-ionic-border-radius};
+  --handle-background: #{$toggle-ionic-handle-background-color};
+  --handle-background-checked: #{$toggle-ionic-handle-background-color};
+  --handle-border-radius: #{$toggle-ionic-handle-border-radius};
+  --handle-height: #{$toggle-ionic-handle-height};
+  --handle-max-height: #{$toggle-ionic-handle-max-height};
+  --handle-width: #{$toggle-ionic-handle-width};
+  --handle-spacing: #{$toggle-ionic-border-width};
+  --handle-transition: #{$toggle-ionic-transition};
+
+  @include globals.typography(globals.$ionic-body-md-regular);
+
+  width: 100%;
+}
+
+// Toggle Wrapper
+// --------------------------------------------------
+
+.toggle-wrapper {
+  align-items: center;
+
+  min-height: globals.$ionic-scale-1200;
+}
+
+// Toggle Native Wrapper
+// ----------------------------------------------------------------
+
+.native-wrapper .toggle-icon {
+  width: globals.$ionic-scale-1100;
+  height: globals.$ionic-scale-600;
+
+  /**
+   * The handle box shadow should not
+   * overflow outside of the track container.
+   */
+  overflow: hidden;
+}
+
+:host(.ion-color.toggle-checked) .toggle-icon {
+  background: current-color(base);
+}
+
+:host(.toggle-activated) .toggle-switch-icon {
+  opacity: 0;
+}
+
+// Toggle Native Wrapper: Focused
+// ----------------------------------------------------------------
+
+:host(.ion-focused) .toggle-icon {
+  outline: globals.$ionic-border-size-050 solid globals.$ionic-state-focus-1;
+  outline-offset: globals.$ionic-border-size-050;
+}
+
+// Toggle Native Wrapper: Pressed
+// ----------------------------------------------------------------
+
+:host(.ion-activated) .toggle-icon::after {
+  @include globals.pressed-state();
+}
+
+// Ionic Toggle: Disabled
+// ----------------------------------------------------------
+
+:host(.toggle-disabled) .toggle-icon::after {
+  @include globals.disabled-state();
+}
+
+// Ionic Toggle Inner Knob: Unchecked
+// ----------------------------------------------------------
+
+.toggle-inner {
+  will-change: background-color, transform;
+
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+
+  color: globals.$ionic-color-neutral-1200;
```

Comment:
No, because this is the icon color that is different from the background

Reply:
I will actually confirm this with UX team.

Reply:
UX just updated the Figma, because it had the wrong color. Now it is updated and the correct color for both on and off iocins are `globals.$ionic-color-neutral-1200`.

Reply:
Thanks for checking!